src/full/Agda/TypeChecking/Unquote.hs

```diff
@@ -753,6 +767,21 @@ evalTCM v = do
       else
         quoteTerm =<< process v
 
+    tcElaborate :: R.PostponedTerm -> R.Type -> TCM Term
+    tcElaborate (R.Postponed {..}) u = do
+      r <- isReconstructed
+      u <- workOnTypes $ locallyReduceAllDefs $ isType_ =<< toAbstract_ u
+      reportSDoc "tc.unquote.elaborate" 50 $ "Elaborating against Type:" <+> pretty u
+      equalType u (El (MetaS postponedSort []) (MetaV postponedType []))
```

Comment:
Dropping the metavariable scope means that `tcElaborate` will never¹ work in a non-empty context (you'd get scope errors for local variables which are part of the solution, but are not in the spine)